CHANGELOG.txt

```diff
@@ -114,4 +114,14 @@
 
 -= Fermion v1.8.1 =-
 
-* Minor release to bugfix #15
\ No newline at end of file
+* Minor release to bugfix #15
+
+-= Fermion v1.9 =-
+
+* Upgrade Electron v13.3.0 -> v21.0.0
+* Upgrade Frida v15.1.3 -> v16.0.5
+* Change Frida TypeScript language bindings to use "@types/frida-gum"
+* Allow Fermion to dynamically fetch the current Frida version on load
+* Fix [..What was the issue here for browser arg parsing..?]
```

Comment:
@NyaMisty can you fill this line in with one sentence to say what the issue was 👍